pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -53,6 +54,9 @@ type stateData struct {
 	// phase for each node
 	// it's initialized in the PreFilter phase
 	podVolumesByNode map[string]*scheduling.PodVolumes
+	// guards map podVolumesByNode in the Filter phase, as the method may be called
+	// in parallel
```

Comment:
We don't need to restrict the protection on `podVolumesByNode as technically it can protect any fields defined here, and also it can be prevent racing in Score().

Reply:
Agree. How about remove this comment?

Reply:
I have modified the comment